analysis_options.yaml

```diff
@@ -4,7 +4,6 @@
 #
 # include: ../../analysis_options.yaml
 #
-include: package:flutter_lints/flutter.yaml
```

Comment:
Why removing the flutter lints include form the root analysis options?

Reply:
It's not found. Since there is no root `pubspec.yaml`, it made sense to me that there are no packages available?

<img width="1136" alt="Screen Shot 2022-09-28 at 11 40 12" src="https://user-images.githubusercontent.com/43643339/192746105-a9693691-c233-490c-864f-de59f98a59ec.png">

Reply:
I think it is used by melos (the dependency is in `melos.yaml`), but not on CI, just when running `melos analyze` locally

Reply:
I put it back. Can file not found be ignored in the analyzer?

Reply:
ah, I did not realize the "other" analyzer complained about it :-p